packages/plugin-langchain/src/LangchainPlugin.ts

```
@@ -0,0 +1,220 @@
+        modelName: this.config.defaultModelName || 'gpt-3.5-turbo', // Default model
+        temperature: 0.7, // Default temperature, can be made configurable
+      };
+
+      // Handle provider-specific configuration
+      switch (this.config.provider.toLowerCase()) {
+        case 'azure':
+          if (!this.config.providerOptions?.azure) {
+            throw new Error(`${this.name}: Azure provider requires providerOptions.azure configuration`);
+          }
+          
+          const azureConfig = this.config.providerOptions.azure;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)